README.md

```diff
@@ -40,8 +40,11 @@ This file contains a number of Scala interview questions that can be used when v
 * What is a companion object?
 * What is the difference between the following terms and types in Scala: `Nil`, `Null`, `None`, `Nothing`?
 * What is `Unit`?
+* What is the difference between a `call-by-value` and `call-by-name` parameter? 
+* How does this benefit to Scala's `Stream` trait?
```

Comment:
Oh, now I understand, It's related with the previous question. My fault, i didn't pay much attention.
Can you put this two questions in the same line? Or make a the second a sub point of the first one.
I prefer to group the related question in some way.